dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -533,7 +534,7 @@ CREATE TABLE `t_ds_process_task_relation` (
   `create_time` datetime NOT NULL COMMENT 'create time',
   `update_time` datetime NOT NULL COMMENT 'update time',
   PRIMARY KEY (`id`),
-  KEY `project_code_process_definition_code_index` (`project_code`,`process_definition_code`) USING BTREE
+  KEY `idx_code` (`project_code`,`process_definition_code`)
```

Comment:
Dose the postgre sql also need to be changed ?

Reply:
Yes，it need